jerry-core/api/jerry.c

```diff
@@ -1037,6 +1037,20 @@ jerry_release_value (jerry_value_t value) /**< API value */
 } /* jerry_release_value */
 
 /**
+ * If the input value is a error value, then return its reference value.
+ * Otherwise, return itself.
+ *
+ * Note:
+ *      returned value must be freed with jerry_release_value, when it is no longer needed.
+ *
+ * @return the real value of the jerry_value
+ */
+jerry_value_t jerry_get_value (jerry_value_t value)
```

Comment:
The name of the newly proposed function does not really reveal what it's doing. IMO. Would `jerry_get_value_without_error_flag` be too long to be useful? Or just `jerry_value_without_error_flag`? (Would align better with existing error-related API functions.)

Reply:
I prefer the `jerry_get_value_without_error_flag`, which is clear because of the `get`